haystack/nodes/reader/farm.py

```diff
@@ -1021,7 +1028,8 @@ def calibrate_confidence_scores(
         Calibrates confidence scores on evaluation documents in the DocumentStore.
 
         :param document_store: DocumentStore containing the evaluation documents
-        :param device: The device on which the tensors should be processed. Choose from "cpu" and "cuda" or use the Reader's device by default.
+        :param device: The device on which the tensors should be processed.
```

Comment:
If we allow `torch.device` and `str` then the docstring should also reflect that. Right now it only mentions the `torch.device`